libs/cases/frontend/view/src/lib/components/case-browse/case-browse.component.ts

```diff
@@ -1,76 +1,193 @@
 })
 export class CaseBrowseComponent {
-    protected readonly _browseFacade = inject(XdBrowseFacadesService);
-    protected readonly _cases = toSignal(this._browseFacade.getAllCases());
-    protected readonly _sortedCases = computed( () => {
-        const cases = this._cases();
-        if (cases === undefined) {
-            return;
-        }
-        const statusOrder = [
-            'OPEN',
-            'INPROGRESS',
-            'OVERDUE',
-            'ONHOLD',
-            'DONE',
-            'CANCELLED',
-            'ARCHIVED',
-        ];
-        const priorityOrder = [ 'EMERGENCY', 'HIGH', 'MEDIUM', 'LOW' ];
-
-        [ ...cases ].sort((a, b) => {
-            const statusAIndex = statusOrder.indexOf(a.status);
-            const statusBIndex = statusOrder.indexOf(b.status);
-
-            if (statusAIndex === statusBIndex) {
-                const priorityAIndex = priorityOrder.indexOf(a.priority.toUpperCase());
-                const priorityBIndex = priorityOrder.indexOf(b.priority.toUpperCase());
-
-                if (priorityAIndex === priorityBIndex) {
-                    return a.id - b.id;
-                } else if (priorityAIndex === -1) {
-                    return 1;
-                } else if (priorityBIndex === -1) {
-                    return -1;
-                }
-                return priorityAIndex - priorityBIndex;
-            } else if (statusAIndex === -1) {
-                return 1;
-            } else if (statusBIndex === -1) {
-                return -1;
-            }
-            return statusAIndex - statusBIndex;
-        });
-
-        return cases;
-    })
-
-    getStatusClasses(_case: ICaseResponse) {
-        return {
-            emergency: _case.priority === 'EMERGENCY',
-            'status-open': _case.status === 'OPEN',
-            'status-inprogress': _case.status === 'INPROGRESS',
-            'status-overdue': _case.status === 'OVERDUE',
-            'status-onhold': _case.status === 'ONHOLD',
-            'status-done': _case.status === 'DONE',
-            'status-cancelled': _case.status === 'CANCELLED',
-            'status-archived': _case.status === 'ARCHIVED'
-        };
-    }
+	protected readonly showPriorityEmergency = signal(true);
+	protected readonly showPriorityHigh = signal(true);
+	protected readonly showPriorityMedium = signal(true);
+	protected readonly showPriorityLow = signal(true);
+
+	protected readonly showTypePlanned = signal(true);
+	protected readonly showTypeIncident = signal(true);
+	protected readonly showTypeAnnotation = signal(true);
+
+	protected readonly showStatusOpen = signal(true);
+	protected readonly showStatusInProgress = signal(false);
+	protected readonly showStatusOverdue = signal(false);
+	protected readonly showStatusOnHold = signal(false);
+	protected readonly showStatusDone = signal(false);
+	protected readonly showStatusCancelled = signal(false);
+	protected readonly showStatusArchived = signal(false);
+
```

Comment:
A lot of signals here. But yeah using an array would be less readable so I think this is okay since it is readable